functions/Test-DbaLastBackup.ps1

```diff
@@ -274,18 +274,19 @@ function Test-DbaLastBackup {
                 $CopyFile = $false
             }
 
-            Write-Message -Level Verbose -Message "Getting recent backup history for $($db.Name) on $instance."
+            Write-Message -Level Verbose -Message "Getting recent backup 
+            for $($db.Name) on $instance."
 
             if (Test-Bound "IgnoreLogBackup") {
                 Write-Message -Level Verbose -Message "Skipping Log backups as requested."
                 $lastbackup = @()
-                $lastbackup += $full = Get-DbaDbBackupHistory -SqlInstance $sourceserver -Database $dbname -IncludeCopyOnly:$IncludeCopyOnly -LastFull -WarningAction SilentlyContinue
-                $diff = Get-DbaDbBackupHistory -SqlInstance $sourceserver -Database $dbname -IncludeCopyOnly:$IncludeCopyOnly -LastDiff -WarningAction SilentlyContinue
+                $lastbackup += $full = Get-DbaDbBackupHistory -SqlInstance $sourceserver -Database $dbname -IncludeCopyOnly:$IncludeCopyOnly -LastFull -DeviceType Disk -WarningAction SilentlyContinue
+                $diff = Get-DbaDbBackupHistory -SqlInstance $sourceserver -Database $dbname -IncludeCopyOnly:$IncludeCopyOnly -LastDiff -DeviceType Disk -WarningAction SilentlyContinue
```

Comment:
I am unsure of other people's use cases or if we can even support them properly. This suits my needs but I don't know if it would break other people's workflows.

What do you think about adding a new parameter and defaulting it to Disk instead? What do you think @wsmelton ?

Reply:
Doesn't bother me, i'm okay with it either way

Reply:
A new parameter can be added but we should not default it to disk. URL backups is a separate device type and would break workflows for those users.

Reply:
Ahh good point, not something I use in my environment today. I'll go ahead and add it as a parameter matching how Get-DbaDbBackupHistory uses it. So by default the functionality wouldn't change, or you could specify an array of types to use.